pom.xml

```diff
@@ -59,7 +59,7 @@
     <sdk.antlr4.dir>${project.build.directory}/eforms-sdk/antlr4</sdk.antlr4.dir>
 
     <!-- Versions - eForms -->
-    <version.eforms-core>1.3.0</version.eforms-core>
+    <version.eforms-core>1.4.0-SNAPSHOT</version.eforms-core>
```

Comment:
A release should not depend on a SNAPSHOT. I guess it's an oversight, because I see no change in eforms-core that would justify it.

Reply:
fixed